tests/shell/shell_recorder/tutorial_wrapper/Test.md

```diff
@@ -0,0 +1,10 @@
+Test
+```sh
+# Mountpoint:/test
+# testcomment
+$ cat /tmp/test
```

Comment:
If we have special characters that say what the commands are, we do not need one for stdout?

To distinguish between stdout/stderr is rather a special case not really needed for tutorials. (At least no special syntax for that). It would be easier if they are merged for the simple syntax.